src/Frontend/config/routes.js

```diff
@@ -10,28 +10,43 @@
 
 'use strict';
 
-import React from 'react';
-import Login from '../components/portal/Login';
-import CreateAccount from '../components/portal/CreateAccount';
-import ActivateAccount from '../components/portal/ActivateAccount';
-import NotFoundPage from '../components/app/layout/NotFoundPage';
-import Application from '../components/app/layout/Application';
-import DashboardIndex from '../components/network/dashboard/Index';
-import Logout from '../components/portal/Logout';
-import { IndexRoute, Router, Route, hashHistory } from 'react-router';
-import { redirectToDashboard, protectPage } from '../util/react/routerHooks';
+import React                                         from 'react';
+// import Login                                      from '../components/portal/Login';
+// import CreateAccount                              from '../components/portal/CreateAccount';
+// import ActivateAccount                            from '../components/portal/ActivateAccount';
+// import NotFoundPage                               from '../components/app/layout/NotFoundPage';
+import Application                                   from '../components/app/layout/Application';
+// import DashboardIndex                             from '../components/network/dashboard/Index';
+// import Logout                                     from '../components/portal/Logout';
+import {IndexRoute, Router, Route, hashHistory}      from 'react-router';
+import {redirectToDashboard, protectPage} from '../util/react/routerHooks';
+import {syncHistoryWithStore}                        from 'react-router-redux';
+import {Provider}                                    from 'react-redux';
+import store                                         from '../config/store';
+
+const HelloWorld = () => {
+  return (
+    <h1>Hello World!</h1>
+  )
+};
+
+//TODO: Change to browesrHistory
+const history = syncHistoryWithStore(hashHistory, store);
 
 export default (
-  <Router history={hashHistory}>
-    <Route component={Application} path="/">
-      <IndexRoute component={Login} onEnter={redirectToDashboard} />
-      <Route component={CreateAccount} path="sign-up" onEnter={redirectToDashboard} />
-      <Route component={ActivateAccount} path="activate/:name/:key" onEnter={redirectToDashboard} />
+  <Provider store={store}>
```

Comment:
not sure if we really need the `Provider` here if we add it in several sub-components (but I'll research about this and leave a comment later here, so no need to change anything here ATM)